edu.wpi.first.wpilib.plugins.cpp/src/main/java/edu/wpi/first/wpilib/plugins/cpp/WPILibCPPPlugin.java

```diff
@@ -81,8 +88,36 @@ public void earlyStartup() {
 		new CPPInstaller(getCurrentVersion()).installIfNecessary(true);
 		Properties props = WPILibCore.getDefault().getProjectProperties(null);
     	WPILibCore.getDefault().saveGlobalProperties(props);
+		updateVariables();
 	}
 
+	@Override
```

Comment:
Why do we need to have an override for a method that just does nothing? Is there some behavior in the regular `startup()` that we want to avoid? If so, please add a comment.